civihr_default_theme/scss/pages/_onboarding-wizard.scss

```diff
@@ -9,4 +9,106 @@
       }
     }
   }
+
+  .form-group {
+    display: inline-block;
+    width: 100%;
+
+    > label {
+      color: $crm-wizard-font-color;
+      display: block;
+      font-size: 13px;
```

Comment:
`font-size: $font-size-base`

Reply:
Value for that variable is 14px, cant use that.